air/src/execution_step/boxed_value/stream.rs

```diff
@@ -223,7 +232,19 @@ impl Stream {
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum Generation {
     Last,
-    Nth(u32),
+    Nth(GenerationIdx),
+}
+
+impl Generation {
+    pub fn last() -> Self {
+        Self::Last
+    }
+
+    pub fn nth(generation_id: u32) -> Self {
```

Comment:
While it is used only in tests, it look suspicious and may be incidentally (ab)used in normal code.

If prefixing with `#[cfg(test)]` works, let's do it.

Reply:
done